src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -64,9 +61,20 @@
     ////////////////////////////////////////////
 
     public ApiClient(ApiClientConfiguration config) {
+        Proxy proxy = ProxySelector.getDefault().select(URI.create(config.getBaseUrl())).iterator().next();
+        URI proxyUri = URI.create(proxy.type() + "://" + proxy.address());
```

Comment:
Ok, can we move this line into the if block at line 76, seems we do not need to create an URI for the DIRECT/SOCKS cases.